executor/builder.go

```diff
@@ -670,6 +681,8 @@ func (b *executorBuilder) buildIndexScan(v *plan.PhysicalIndexScan) Executor {
 		aggregate:            v.Aggregated,
 		aggFuncs:             v.AggFuncsPB,
 		byItems:              v.GbyItemsPB,
+		needColHandle:        v.NeedColHandle,
```

Comment:
I think `needColHandle` is redundant. We can check if `handleCol` is nil instead.